### ISSUE TITLE
updatePassword

### ISSUE THREAD

**Issue by jcaruso001** [step-up-labs/firebase-authentication-dotnet#79]
Is updatePassword not supported here?

```
var user = firebase.auth().currentUser;
var newPassword = getASecureRandomPassword();

user.updatePassword(newPassword).then(function() {
  // Update successful.
}).catch(function(error) {
  // An error happened.
});
```

**Comment by bezysoftware** [comment]
No. But PR's are welcome

**Comment by bezysoftware** [comment]
Resolves in #92
